setup.py

```diff
@@ -32,6 +32,7 @@ def read(fname):
         "numpy==1.16",
         "pandas==0.25.3",
         "pyfftw",
+        "PyWavelets",
```

Comment:
Noting that we already brought this dependency in our tree of depends, but I made it explicit here since we're now explicitly importing it.

Reply:
You mean that it was installed by one of our dependencies?

Reply:
Yes, at least one (not sure which, but I could take some guesses...).  This is just to make sure if the other depends move/go away that we still bring it in.  I didn't sense any strong dependence on version here.

Reply:
Great. Just wanted to check.